LocalSettings.php

```diff
@@ -3684,7 +3685,9 @@
 	
 	// Rights
 	'+wgAvailableRights' => [
-		'default' => [],
+		'default' => [
+			'watched-person',
```

Comment:
Others then need moving to + codes. I wonder if there's a way to just add straight to the global group.

Reply:
If it's in `$wgAvailableRights` for `default`, it should be possible to then just add that user right in Special:GlobalGroupPermissions, I believe